playbook.yaml

```
@@ -0,0 +1,30 @@
+---
+- name: hadoop-testing playbook
+
+  hosts: local
+
+  pre_tasks:
+    - name: orchestrate download file
+      ansible.builtin.template:
+       src: templates/env.j2
+       dest: "{{ playbook_dir }}/.env"
+
+    - name: Changing perm of ".env", adding "+x"
+      file: dest="{{ playbook_dir }}/.env" mode=a+x
+
+  tasks:
+    - name: orchestrate download file
+      ansible.builtin.template:
+       src: templates/download.j2
```

let's keep the filename with exact `<dest_fileaname>.j2`
```suggestion
       src: templates/download.sh.j2
```

---

OK